### PR TITLE
Improve Prompts & Commands list UI 

### PR DESCRIPTION
**Pull request by vovakulikov**
This is based on @sqs PR https://github.com/sourcegraph/cody/pull/5418

| Before | After |
| ------ | ------ |
| Screenshot 2024-09-02 at 20 25 42 | Screenshot 2024-09-02 at 20 24 49 |

This PR
- Adds new Prompt UI to the welcome message (the same items - prompts and commands but with new styles, less noisy) 
- Fixes problem with the "Scroll to bottom" button on the welcome message screen when a user hasn't submitted any message 
- Fixes problem with setting prompt with prompts tabs
- Removes Prompts Select component from the toolbar 

## Test plan
- Check VSCode Cody UI, and make sure that you can use prompts and commands (from the welcome screen and from the prompts tab)
- Check Cody Web demo and make sure that you can see only prompts

### CODE REVIEW

File: web/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.7.6
+- Simplify prompt list UI (no prompt toolbar anymore)
```
Comment (sqs) [inline web/CHANGELOG.md:2]:
```suggestion
- The "Prompts" toolbar item in chat is no longer displayed. To use a prompt from the prompt library, select it from the list shown on the chat tab or prompts tab.
```

to use full descriptive sentences

File: web/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+- Fix setting prompt from prompts tab
```
Comment (sqs) [inline web/CHANGELOG.md:3]:
```suggestion
```

This just adds noise to the changelog IMO, so remove it (the `Fix setting prompt...`).

**Review by sqs — APPROVED** [review]

**Review by valerybugakov — APPROVED** [review]
Tested locally ✅

When switching accounts from sg02 to DotCom the list of prompts was not updated until I reopened the sidebar.

Would it be nice to have a better contrast for the draft badge similar to the dark theme?
Screenshot 2024-09-03 at 10 42 46

**Comment by vovakulikov** [comment]
Thanks for the review folks

> When switching accounts from sg02 to DotCom, the list of prompts was not updated until I reopened the sidebar.

Yeah, this problem exists on the main as well; I will take a look after the release. It's not a problem for Cody Web, so it's not a blocker for the Sourcegraph release, but I will look into this before the next VSCode release. 

> Would it be nice to have a better contrast for the draft badge similar to the dark theme?

Yeah, I recently slightly updated them (didn't touche the colors though) we have plans to revamp the whole design kit for chat UI, so it will be addressed shortly

**Comment by kalanchan** [comment]
@sqs @taylorsperry just a heads up - this won't be included in the Sept 4 JetBrains release, we already cut the final version and is waiting for marketplace approval. We'll follow up after Sept 4 to get this in

**Pull request merged**